src/app/core/sidenav/sidenav.component.html

```diff
@@ -23,11 +19,6 @@ <h6 class="m-b-10">
 
   <div class="useful-links-wrapper">
     <hr>
-    <p class="link-with-icon" *ngIf="canCreateNewRecord" (click)="createNewRecord()">
-      <mat-icon>queue</mat-icon>
-      <span>Create New Record</span>
-    </p>
-
     <p class="link-with-icon" *ngIf="createNewEntityLabel" (click)="openAddNewOptionDialog()">
```

Comment:
Are we going to keep this link as part of the sidenav?

Reply:
@DAquilinaCinchy Keep the "create" option on the sidenav? No. It will only appear in the new actions dropdown.

Reply:
This is moved to the menu as 'Create'

Reply:
@cinchy-m I agree, I was just wondering why it hadn't been removed, but given our call to clarify what it actually does I can see why it hasn't been moved yet.